website/content/docs/site-generator-overview.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Overview
+group: Guides
+weight: 1
+---
+
+A common pain point with static site generators is that they are hard to use for non-technical editors. Content and source code are stored together in the same Git repository. Making changes to the site content requires editors to go through a sequence of steps:
```

Comment:
I believe this part until `Adding Netlify CMS to your site` has a very similar intention to the [general overview](https://www.netlifycms.org/docs/intro/), so we might be able to combine the two

Reply:
What do you suggest? Move this part to the [general overview](https://www.netlifycms.org/docs/intro/), or delete it?